app/models/wrate.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const Wrate = sequelize.define(
+    'Wrate',
```

Comment:
I should change the name to "Califications" (just a preference)

Reply:
I changed to ratings acording to ratingUserId field required in trello card